website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -64,7 +64,9 @@ python example/dummy_training.py -m
 
 You can also override the search space parametrization:
 ```bash
-python example/dummy_training.py -m db=mnist,cifar batch_size=4,8,16 lr=log:0.001:1 dropout=0:1
+python example/dummy_training.py -m \
```

Comment:
can  you also rename dummy_training to my_app.py to be consistent with the rest of the examples?

Reply:
done :)